sdk/src/Services/S3/Custom/Transfer/Internal/DownloadDirectoryCommand.cs

```diff
@@ -98,7 +98,10 @@ private TransferUtilityDownloadRequest ConstructTransferUtilityDownloadRequest(S
             downloadRequest.Key = s3Object.Key;
             var file = s3Object.Key.Substring(prefixLength).Replace('/', Path.DirectorySeparatorChar);
             downloadRequest.FilePath = Path.Combine(this._request.LocalDirectory, file);
-                        
+            downloadRequest.ServerSideEncryptionCustomerMethod = this._request.ServerSideEncryptionCustomerMethod;
```

Comment:
Could you add a new integration test here? https://github.com/aws/aws-sdk-net/blob/main/sdk/test/Services/S3/IntegrationTests/TransferUtilityTests.cs

Here's one example for uploading objects (to a bucket that requires `SSE-C`): https://github.com/aws/aws-sdk-net/blob/3.7.729.0/sdk/test/Services/S3/IntegrationTests/TransferUtilityTests.cs#L438-L464